esp-hal/MIGRATING-0.22.md

```diff
@@ -466,6 +478,17 @@ The Address and Command enums have similarly had their variants changed from e.g
 + Command::_1Bit
 ```
 
+`write_byte` and `read_byte` were removed and `write_bytes` and `read_bytes` can be used as replacement.
+
+e.g.
+
+```rust
+let mut byte = [0u8; 1];
+spi.read_bytes(&mut byte);
+```
+
+`driver.write_bytes(&[data])?` and `driver.read_bytes(&mut buffer)?`
```

Comment:
What about them?

Reply:
Forgot to remove this line, removed now.